src/ResourceManager/Common/Commands.Common.Strategies/Compute/ComputeStrategy.cs

```diff
@@ -0,0 +1,24 @@
+﻿using Microsoft.Azure.Management.Compute;
+using Microsoft.Azure.Management.Compute.Models;
+using System;
+using System.Threading.Tasks;
+
+namespace Microsoft.Azure.Commands.Common.Strategies.Compute
+{
+    public static class ComputePolicy
+    {
+        public static ResourceStrategy<Model> Create<Model, Operations>(
```

Comment:
These should be `TModel` and `TOperations`, by convention.

Reply:
This applies everywhere.

Reply:
Done.